freertos/include/FreeRTOS.h

```diff
@@ -125,7 +125,7 @@ extern "C" {
 #endif
 
 #ifndef INCLUDE_vTaskSuspend
-#define INCLUDE_vTaskSuspend 0
+#define INCLUDE_vTaskSuspend 1
```

Comment:
FreeRTOS.h holds the default values for these definitions (that's why there's an `ifndef`) so we usually never touch this file. The application-specific settings are in FreeRTOSConfig.h. There, you'll see that vTaskSuspend is already enabled.

Since FreeRTOS.h includes FreeRTOSConfig.h, if something is defined in the config file, it won't be redefined here due to the ifndef.